metabase/metabase
# Show/hide card visualization for pinned collection items

Epic https://github.com/metabase/metabase/issues/23222

Adds a way to hide visualization for pinned questions in collections. Please note that the backend is not implemented yet and hiding visualization from the menu will do nothing.

How to test:
- Change https://github.com/metabase/metabase/blob/38e9a68b49a4102952de13328b707c057357c50d/frontend/src/metabase/collections/utils.ts#L85 to `item.description != null`
- Open a collection with questions of different visualization types. Questions with descriptions should be visualized, others should be listed at the bottom with static images.
- `Show/Don't show visualization` menu option should not be available for dashboards and non-pinned collection items

<img width="1303" alt="Screenshot 2022-06-14 at 17 06 43" src="https://user-images.githubusercontent.com/8542534/173597723-d091294a-0b62-448b-8ecc-7e60ac90d158.png">
<img width="1324" alt="Screenshot 2022-06-14 at 17 07 15" src="https://user-images.githubusercontent.com/8542534/173597749-6849cbf5-72c2-49e4-9178-7d2268b4a9c1.png">
<img width="1322" alt="Screenshot 2022-06-14 at 17 07 20" src="https://user-images.githubusercontent.com/8542534/173597757-e3a02beb-9745-4ace-a81b-b54b1c2caee8.png">
<img width="700" alt="Screenshot 2022-06-14 at 19 16 32" src="https://user-images.githubusercontent.com/8542534/173626323-9d641ebe-3090-45e9-84ce-3369c2db655e.png">
<img width="693" alt="Screenshot 2022-06-14 at 19 16 38" src="https://user-images.githubusercontent.com/8542534/173626330-7d84ae26-56d4-4f8a-b029-569383110623.png">
